src/web/views/phone-number.njk

```diff
@@ -13,7 +13,7 @@
     }
   }) %}
 
-    <span class="govuk-hint govuk-!-margin-bottom-6">{{ formDescription }}</p>
+    <span class="govuk-hint govuk-!-margin-bottom-6">{{ formDescription }}</span>
```

Comment:
As this is used in a number of places, should we create a Nunjucks macro, `macros/htbhf-hint.njk`?

Reply:
Now using govukHint macro